e2e-tests/specs/relay-home-page.spec.ts

```diff
@@ -14,46 +14,36 @@ test.describe('Firefox Relay - Landing Page - Visual Regression', () => {
       defaultScreenshotOpts
     );
   });
-
-  test('Verify firefox apps and service', async ({ landingPage }) => {    
-    await landingPage.openFirefoxAppsServices()
-    await expect(landingPage.firefoxAppsServicesExpanded).toHaveScreenshot(
-      `${process.env.E2E_TEST_ENV}-firefoxAppsServicesExpanded.png`,
-      defaultScreenshotOpts
-    );
-  })  
 });
 
 
 test.describe('Check header buttons and their redirects,  C1812638',  () => {
   test.beforeEach(async ({ landingPage }) => {
     await landingPage.open()
-  });  
+  });
 
-  test('Verify home FAQ button redirect', async ({ landingPage, page }) => {
-    await landingPage.goToFAQ() 
-    expect(page.url()).toContain(`${process.env.E2E_TEST_BASE_URL}/faq/`)
+  test('Verify home FAQ button redirect', async ({ landingPage }) => {
+    const FAQRedirectLink = await landingPage.FAQButton.getAttribute('href')
+    expect(FAQRedirectLink).toEqual("/faq/")
```

Comment:
nit: we seem to be mixing single quotes and double quotes in these files. Yeah, super nit, but 👁️.
Guessing that means we aren't running ESLint or Prettier on these files (or else we haven't configured ESLint to enforce a quote style).